fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -32,11 +32,17 @@ source "${HCPPIPEDIR}/global/scripts/fsl_version.shlib"          # Functions for
 
 FIELDMAP_METHOD_OPT="FIELDMAP"
 SIEMENS_METHOD_OPT="SiemensFieldMap"
-GENERAL_ELECTRIC_METHOD_OPT="GeneralElectricFieldMap"
+# For GE HealthCare Fieldmap Distortion Correction methods 
+# see explanations in global/scripts/FieldMapPreprocessingAll.sh
+GENERAL_ELECTRIC_METHOD_OPT="GEHealthCareLegacyFieldMap" 
+GE_HEALTHCARE_METHOD_OPT="GEHealthCareFieldMap"
 PHILIPS_METHOD_OPT="PhilipsFieldMap"
 SPIN_ECHO_METHOD_OPT="TOPUP"
 NONE_METHOD_OPT="NONE"
 
+# Minimum required FSL version for GENERAL_ELECTRIC_METHOD_OPT and GE_HEALTHCARE_METHOD_OPT
```

Comment:
Do we need the duplication of checking FSL version for GE where, when, from what I can tell, the same version checks are already built into `FieldMapPreprocessingAll.sh`?

Reply:
It also means there are two places in the code making an identical check, and we'd have to remember to change both in any future changes.  @coalsont : How do you want to handle this sort of thing?